libnettracer/src/netstat.h

```diff
@@ -57,8 +57,10 @@ class NetStat {
 	bool add_header_mode_ = false;
 	bpf::BPFMapsWrapper* mapsWrapper;
 	std::ostream* os;
+	int field_width = 22;
```

Comment:
Here the field_width is not const but in another place (netstat.cpp:322) 22 is a literal. That may lead to some discrepancies if something changes the width in the class